tmt-app/src/Components/BobaPath.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {Container, Typography} from '@mui/material';
+import ForwardArrow from '../Images/Forward.png';
+
+
+function BobaPath(){
+    return(
+        <Container>
+            <Container 
```

Comment:
Should this just be a `Box` component? Is there an intention to nesting `Container` components?

Reply:
Yeah it should be a single Container and I used Container over Box because of the preset styling